core/src/avm2/globals/avmplus.rs

```diff
@@ -355,6 +355,19 @@ fn describe_internal_body<'gc>(
                 let method = vtable
                     .get_full_method(*disp_id)
                     .unwrap_or_else(|| panic!("Missing method for id {disp_id:?}"));
+
+                // Don't include methods that also exist in any interface
+                let mut is_interface = false;
+                for interface in method.class.all_interfaces().iter() {
```

Comment:
I think `iter().any(|interface| .. )` would be a bit cleaner instead of a mutable variable.